homeworks/ekaterina.sharova_rogerosha/PokemonProject/pokemons.html

```diff
@@ -0,0 +1,322 @@
+<!doctype html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport"
+            content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+        <meta http-equiv="X-UA-Compatible" content="ie=edge">
+        <title>PokemonProject: Pokemons</title>
+        <link
+            href="./pokemons.css"
+            rel="stylesheet"
+            type="text/css">
+    </head>
+    <body>
+        <header class="upper-header">
+            <div class="ma-logo">
+                <img class="MAlogo" src="images/MA-logo.svg" alt="Masters Academy">
+            </div>
+            <ul class="navigation">
+                <li>
+                    <a href="#" class="navigation-item pokemons">Pokemons</a>
+                </li>
+                <li>
+                    <a href="./index.html" class="navigation-item about">About</a>
+                </li>
+            </ul>
+        </header>
+        <div class="content">
+            <h1 class="headline">Study Project "Pokemons"</h1>
+            <div class="card-container">
+                <section class="card">
+                    <div class="card-header">
+                        <img src="images/Urshifu.svg" alt="Urshifu" class="card-image">
```

Comment:
Same here, you've even exported it [pokemoncard.png](https://github.com/MastersAcademy/fe-course-basic-2023/pull/154/files#diff-0805de797b66e11460493aededec4ae0866b032bfdcaad8aec3692f6c23c689a)

Reply:
My bad, everything is good

![image](https://github.com/MastersAcademy/fe-course-basic-2023/assets/71977925/1f505fc1-816e-4618-8753-c3fbefc61e73)